tokio/src/io/read_buf.rs

```diff
@@ -284,7 +284,10 @@ unsafe impl<'a> BufMut for ReadBuf<'a> {
     }
 
     fn chunk_mut(&mut self) -> &mut UninitSlice {
-        unsafe { UninitSlice::from_raw_parts_mut(10 as *mut u8, 10) }
+        let ptr = 10 as *mut u8; // any ideas?
```

Comment:
I guess [`unfilled_mut()`](https://docs.rs/tokio/latest/tokio/io/struct.ReadBuf.html#method.unfilled_mut)?
Also because of something I read somewhere while spying on PRs like this I think it might be necessary to read the `len` before getting the pointer.

Reply:
![image](https://user-images.githubusercontent.com/83634595/229120076-ef9068cb-4f81-4223-82b8-9b65b8af1e0b.png)